README.md

```diff
@@ -0,0 +1,39 @@
+# PrxAccess
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/prx_access`. To experiment with that code, run `bin/console` for an interactive prompt.
```

Comment:
This could use some love to make it more specific - just bare bones, this gem is for accessing prx apis